dcpy/library/templates/dcp_waterfront_access_map_pow.yml

```diff
@@ -0,0 +1,33 @@
+dataset:
+  name: dcp_waterfront_access_map_pow
+  acl: public-read
+  source:
+    arcgis_feature_server:
+      server: nyc
```

Comment:
Looking a little more at this server, it seems like it might specifically be dcp?

https://services5.arcgis.com/GfwWNkhOj9bNBqoJ/ArcGIS/rest/services